x/qgb/abci.go

```diff
@@ -2,23 +2,31 @@ package qgb
 
 import (
 	"errors"
+	"time"
 
 	sdkerrors "cosmossdk.io/errors"
+
 	"github.com/celestiaorg/celestia-app/x/qgb/keeper"
 	"github.com/celestiaorg/celestia-app/x/qgb/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// SignificantPowerDifferenceThreshold the threshold of change in the validator set power
-// that would need the creation of a new valset request.
-const SignificantPowerDifferenceThreshold = 0.05
+const (
+	// SignificantPowerDifferenceThreshold is the threshold of change in the validator set power
+	// that would trigger the creation of a new valset request.
+	SignificantPowerDifferenceThreshold = 0.05
+
+	// AttestationExpiryTime the expiration time of an attestation after which it will be pruned.
+	AttestationExpiryTime = 3 * 7 * 24 * time.Hour
```

Comment:
[nit] this can be improved per https://github.com/celestiaorg/celestia-app/pull/1799#discussion_r1202575777

Reply:
https://github.com/celestiaorg/celestia-app/pull/1822